store/copr/coprocessor.go

```diff
@@ -858,6 +858,9 @@ func (sender *copIteratorTaskSender) run() {
 		if exit {
 			break
 		}
+		if connID > 0 {
```

Comment:
Can use `RequestSource` to specify paused cop tasks instead?

Reply:
It should be ok,  adding `connID` is helpful to group requests by session in debug logs.

Reply:
OK, LGTM